### ISSUE TITLE
ALM Toolkit not opening

### ISSUE THREAD

**Issue by floari** [microsoft/Analysis-Services#65]
Hi,

I cant get ALM Toolkit to work. Clicking on the .exe does nothing.
Is this a known problem?

Thanks!

**Comment by christianwade** [comment]
Hi, no this is not known. Thanks for reporting. It's possibly something to do with the environment. Perhaps permissions needed are not available. Are you admin on the machine? Does it work on other machines?

**Comment by floari** [comment]
Hi,

yes I am admin on the machine. I reinstalled ALM toolkit and now powerbi prompts "unable to launch external tool - executable not found" - if I navigate to the correctly installed .exe in windows the tool does not open either.

Can I share any logs which might help?

Thanks!

**Comment by christianwade** [comment]
Hi, does it work if you launch ALM Toolkit from the start menu?
Do you have a file called **almtoolkit.pbitool.json** in a folder such as this?
C:\Program Files (x86)\Common Files\Microsoft Shared\Power BI Desktop\External Tools

**Comment by floari** [comment]
Hi,

yes, the file is located within this filepath. Here are the first content lines:

`"version": "1.0",
    "name": "ALM Toolkit",
	"description": "Use ALM Toolkit to perform easy incremental deployment, code/repo/branch merging, and reuse definitions between datasets.",
	"path": "C:\\Program Files\\Power BI ALM Toolkit\\Power BI ALM Toolkit\\AlmToolkit.exe",
	"arguments": "\"%server%\" \"%database%\"",`

ALM Toolkit does not start either if I open it from the startmenu or directly via the .exe file.

**Comment by christianwade** [comment]
Yeah I believe it's associated with the chromium components.
Can you please download and install C++ Redist 2013 x86 and see if that fixes it?
https://www.microsoft.com/en-us/download/details.aspx?id=40784

**Comment by floari** [comment]
Hi,

unfortunately this does not fix the problem.
We checked the tool on our colleagues laptop and there it works. We do have standard configured Windows 10 workstations (although I have the new Lenovo ThinkPad T14i - colleagues are using T495). Software should be the same.

Thanks,
Florian

**Comment by floari** [comment]
Update: I installed vcredist 2013 and 2015-2019 each x46 and x86 and now it works! 

Thanks for your support!

**Comment by christianwade** [comment]
Good news!

**Comment by shayeny** [comment]
Had the same issue. Solved by installing both the C++ Redist 2013 x86 & x64, through the link that Christian provided. 💯

**Comment by CavPeng** [comment]
Hello,
solution does not work on my laptop which already installed  C++ Redist 2012,2013,2015-2019 (X64)/(X86)
sad...

**Comment by Geeta712** [comment]
Hello,
  I am facing the same issue. ALM toolkit is not working on my system also not working in power BI external tools. I followed the same procedure I have installed the Microsoft visual C++ 2015-2022 Redistributable (x64) & (x86).
still not working. Please provide the solution. 
Thanks in advance.

**Comment by christianwade** [comment]
Hi, did you also use a custom installation path?

**Comment by Geeta712** [comment]
I have used the following link for Microsoft visual C++ 2015-2022 Redistributable (x64) & (x86)
https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170
& for ALM toolkit I have installed the latest version  from ALM toolkit site.

**Comment by christianwade** [comment]
Got it. And did you install ALM Toolkit to the default path? Something like this: C:\Program Files\Power BI ALM Toolkit\Power BI ALM Toolkit

**Comment by Geeta712** [comment]
Yes @christianwade, I have installed ALM toolkit to the default path. path is  C:\Program Files\Power BI ALM Toolkit\Power BI ALM Toolkit

**Comment by christianwade** [comment]
Hi, if you have installed the C++ redist, then I can only assume it is an environmental issue. Without a repro to debug, not sure if you can try a different machine. We will do a digital cert soon (timeline not confirmed) and should retry as hopefully resolves.

**Comment by Geeta712** [comment]
Yes, may be it's environmental issue.  Okay, thank you for replying my queries.
Thanks :)

**Comment by devgabrielfaria** [comment]
Oi também não estou conseguindo instalar o ALM no Windows 11 podem me ajudar ?

**Comment by christianwade** [comment]
Oi, você instalou esse C++ Redist 2013 x86?
https://www.microsoft.com/en-us/download/details.aspx?id=40784
